ggshield/git_shell.py

```diff
@@ -58,7 +62,10 @@ def get_list_commit_SHA(commit_range: str) -> List[str]:
     :param commit_range: A range of commits (ORIGIN...HEAD)
     """
 
-    return shell([GIT_PATH, "rev-list", "--reverse", commit_range])
+    commit_list = shell([GIT_PATH, "rev-list", "--reverse", *commit_range.split()])
+    if "" in commit_list:
+        commit_list.remove("")
```

Comment:
:question: `remove` only removes one occurrence, the empty string can only appear once? (I think I remember it's a trailing new line that cause it)

Reply:
Only happens when the `git rev-list` returns an empty rev-list but no error example `git rev-list HEAD..HEAD`